.NET/Microsoft.Recognizers.Definitions.Common/English/NumbersDefinitions.cs

```diff
@@ -55,7 +55,7 @@ public static class NumbersDefinitions
       public static readonly string OrdinalRoundNumberRegex = $@"(?<!an?\s+){RoundNumberOrdinalRegex}";
       public static readonly string OrdinalEnglishRegex = $@"(?<=\b){AllOrdinalRegex}(?=\b)";
       public const string FractionNotationWithSpacesRegex = @"(((?<=\W|^)-\s*)|(?<=\b))\d+\s+\d+[/]\d+(?=(\b[^/]|$))";
-      public const string FractionNotationRegex = @"(((?<=\W|^)-\s*)|(?<![/-])(?<=\b))\d+[/]\d+(?=(\b[^/]|$))";
+      public const string FractionNotationRegex = @"((((?<=\W|^)-\s*)|(?<![/-])(?<=\b))\d+[/]\d+(?=(\b[^/]|$))|[\u00BC-\u00BE\u2150-\u215E])";

```

Comment:
It seems this regex is the same everywhere, right? It would be better to move it to BaseNumber.yaml for it to be reused by all languages.

Reply:
Moved regex to BaseNumber.yaml and added test case to other languages.